### PR TITLE
Fix: Remove unused argument

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] removes an unused argument

Somewhat related to #477.

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 
 $finder = new PhpCsFixer\Finder();
-$config = new PhpCsFixer\Config('json-schema', 'json-schema style guide');
```
Comment (localheinz) [inline .php_cs.dist]: The `$description` argument was removed in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2244.

Comment (erayd) [reply .php_cs.dist]: Why have you also removed `$name`, when only `$description` has been deprecated?

File: .php_cs.dist [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+$config = new PhpCsFixer\Config('json-schema');
```
Comment (localheinz) [inline .php_cs.dist]: The name is still here, @erayd.

Comment (erayd) [reply .php_cs.dist]: My mistake, sorry! I looked at the diff and thought I saw the whole line removed.

Comment (localheinz) [reply .php_cs.dist]: All good, @erayd, thank you!

**Comment by localheinz** [comment]
Thank you, @bighappyface and @erayd!

**Pull request merged**